modules/issue_tracker/test/Issue_TrackerTest.php

```diff
@@ -154,17 +143,17 @@ private function _testFilter($name,$value)
      */
     function testClearFormIssueTracker()
     {
-         $this->safeGet($this->url . "/issue_tracker/");
-         $keywordElement = $this->webDriver->findElement(
-             WebDriverBy::Name("keyword")
-         );
-         $keywordElement->sendkeys('TestTestTest');
-         //click clear form button
-         $this->webDriver->findElement(WebDriverBy::Name("reset"))->click();
-         $bodyText =$this->webDriver->findElement(
-             WebDriverBy::Name("keyword")
-         )->getText();
-         $this->assertNotContains("TestTestTest", $bodyText);
+         //$this->safeGet($this->url . "/issue_tracker/");
+         //$keywordElement = $this->webDriver->findElement(
+         //    WebDriverBy::Name("keyword")
+         //);
+         //$keywordElement->sendkeys('TestTestTest');
+         ////click clear form button
+         //$this->webDriver->findElement(WebDriverBy::Name("reset"))->click();
+         //$bodyText =$this->webDriver->findElement(
+         //    WebDriverBy::Name("keyword")
+         //)->getText();
+         //$this->assertNotContains("TestTestTest", $bodyText);
```

Comment:
Why is this test commented out?

Reply:
Not sure. @kongtiaowang do you have any idea?

Reply:
The 'Clear Form' button is now gone and has become the 'Clear Filter' button.